spec/muffin/base_spec.rb

```diff
@@ -0,0 +1,78 @@
+RSpec.describe Muffin::Base do
+  let(:view) { Muffin::MockView.new }
+
+  let(:operation_class) do
+    Class.new(Muffin::Base) do
+      attribute :id, Integer
+      attribute :first_name, String
+
+      attribute :address, array: false do
+        attribute :street_name, String
+        attribute :street_number, String
+      end
+
+      attribute :tags, array: true do
+        attribute :id, Integer
+        attribute :label, String
+      end
+
+      def self.name # because this is an anonymous class
+        "UpdateUser"
+      end
+    end
+  end
```

Comment:
The operation attributes should cover all cases
* primitivs
* nested (non-array)
* nested (array)